src/api/localStorage.js

```diff
@@ -0,0 +1,110 @@
+export class LocalStorage {
+  constructor() {
+    this.users = JSON.parse(localStorage.getItem('UserAccounts')) || [];
+  }
+
+  createUser(user) {
+    this.updateUsers();
+    user.id = this.generateId();
+    this.users.push(user);
+    localStorage.setItem('UserAccounts', JSON.stringify(this.users));
+    console.log('User Added: ', user);
+    return this.users;
+  } 
+
+  updateUsers() {
+    this.users = JSON.parse(localStorage.getItem('UserAccounts')) || [];
+    return this.users;
+  }
+ 
+  getUser(id) {
+    const user = this.findUserMatch(id);
+    console.log('getUser() -> User Found: ', user);
+    return user;
+  }
+
+  getAllUsers() {
+    console.log('getAllUsers() -> Users Found: ', this.users);
+    return this.users;
+  }
+
+  editUser(id, editName, editValue) {
+    const editObj = {};
+    editObj[`${editName}`] = `${editValue}`;
+    const updatedUser = Object.assign(this.getUser(id), editObj);
+    console.log('Updated User: ', updatedUser);
+    this.createUser(updatedUser);
+  }
+
+  replaceUser(id, update) {
+    const index = this.getUserIndex(id);
+    const replacedUser = Object.assign(
```

Comment:
Could you just put this on a single line?
```
   const replacedUser = Object.assign(this.users[index], update);
```

Reply:
Used that same Array.find here to get my user and just passed that in here too:
    const user = this.users.find(user => String(user.id) === String(id));
    const replacedUser = Object.assign(user, update);